### PR TITLE
fix(audio): swap non-phaser sounds to use capacitor audio

### PR DESCRIPTION
**Pull request by PaulPaulDevelops**
# Pull request type

-   [X] Bugfix
-   [X] Feature
-   [ ] Code style update (formatting, renaming)
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] Documentation content changes
-   [ ] E2E Test(s)
-   [ ] Other (please describe):

# What is the current behavior?
- After the success/fail sounds, all audio in the application would not work as expected.

# What is the new behavior?
Did some debugging and found that these specific sounds were not set by Phaser, but instead were HTMLAudioElements. I looked to see if we could keep these and add a pause call (in case the audio is still in the foreground after playing / navigating away), but that did not work in this case.

I migrated these audio sounds to use Capacitor Native Audio. I pre-loaded up the sounds in the world (where we do phaser loading) for consistency.
I renamed the two sound folders in our assets to be more specific on which sounds are set/used by phaser vs. what will be copied into the ios/android folders on cap sync/copy.

# Does this introduce a breaking change?

-   [ ] Yes
-   [X] No

# Other information

### CODE REVIEW

File: libs/shared/phaser-singleton/src/lib/scenes/world.scene.ts [hunk 1]
```diff
@@ -106,8 +107,20 @@ export class WorldScene extends Phaser.Scene {
             this.load.image(MUTE_BUTTON, 'assets/buttons/mute.png');
 
             // * Loading audio files
-            this.load.audio(BACKGROUND_AUDIO_KEY, 'assets/audios/background/background-music-for-mobile-casual-video-game-short-8-bit-music-164703.mp3');
-            this.load.audio(JUMP_AUDIO_KEY, 'assets/audios/jump/cartoon-jump-6462.mp3');
+            this.load.audio(BACKGROUND_AUDIO_KEY, 'assets/phaser-audios/background/background-music-for-mobile-casual-video-game-short-8-bit-music-164703.mp3');
+            this.load.audio(JUMP_AUDIO_KEY, 'assets/phaser-audios/jump/cartoon-jump-6462.mp3');
+            void NativeAudio.preload({
+                assetId: 'fail',
```
Comment (Ricardo385) [inline libs/shared/phaser-singleton/src/lib/scenes/world.scene.ts]: Hey, could we have this `assetId` as a constant?

File: libs/shared/phaser-singleton/src/lib/scenes/world.scene.ts [hunk 2]
```diff
@@ -106,8 +107,20 @@ export class WorldScene extends Phaser.Scene {
...
+                assetPath: 'public/assets/capacitor-sounds/failure-drum-sound-effect-2-7184.mp3',
+                audioChannelNum: 1,
+                isUrl: false,
+            });
+            void NativeAudio.preload({
+                assetId: 'success',
```
Comment (Ricardo385) [inline libs/shared/phaser-singleton/src/lib/scenes/world.scene.ts]: Same here, could we have this `assetId` as a constant?

**Review by Ricardo385 — CHANGES_REQUESTED** [review]
Nice and great work here! Just a minor comment

**Comment by PaulPaulDevelops** [comment]
@Ricardo385 @emmanuelOpenForge thank you for the feedback! It has been resolved.

**Review by Ricardo385 — APPROVED** [review]

**Pull request merged**